sw/host/opentitanlib/src/dif/lc_ctrl.rs

```diff
@@ -39,6 +39,10 @@ with_unknown! {
 }
 
 impl DifLcCtrlState {
+    pub fn from_redundant_encoding(encoding: u32) -> Result<Self> {
+        Ok(DifLcCtrlState(encoding & 0x1fu32))
```

Comment:
Would you mind adding a `// TODO: check the upper bits of the encoding` here?

Reply:
do you mean to ensure its a redundantly encoded lc state? does it matter? if its not redundantly encoded it will still produce the correct result no?

Reply:
It's kind of a defence-in-depth approach. If it were not redundancy encoded, then bail because something went wrong.

Reply:
since this is close to passing CI and I need to get it in asap, I added this to the follow on pr (#20288)